packages/kbn-optimizer/limits.yml

```diff
@@ -101,7 +101,7 @@ pageLoadAssetSize:
   visTypeMetric: 23332
   bfetch: 22837
   kibanaUtils: 79713
-  dataViews: 43532
+  dataViews: 84552
```

Comment:
Any possibility of asynchronously loading the usage here to avoid the increase in page load?

Reply:
This PR increases the page load bundle size by 13 bits... This is a massive increase to the limit and I think it should come along with an explanation.

Reply:
no longer importing the code that increased the bundle size.